cpp/modmesh/buffer/pymod/wrap_SimpleArray.cpp

```diff
@@ -143,7 +143,7 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArray
                                    { return pybind11::cast(SimpleArrayPlex(arr)); })
             .wrap_modifiers()
             .wrap_calculators()
-            //
+            // ATTENTION: always keep the same interface between WrapSimpleArrayPlex and WrapSimpleArray
```

Comment:
I love this.  Useful comment.  Sweet.

In the future, the words will be turned into a structure/pattern by the power of open source.